era5cli/fetch.py

```diff
@@ -250,6 +250,12 @@ def _split_variable_yr(self):
 
     def _product_type(self):
         """Construct the product type name from the options."""
+        if self.land and self.ensemble:
+            raise ValueError(
```

Comment:
Sorry, I got busy and forgot about that! I was planning on adding an `AssertionError`, not a `ValueError`. The idea behind that was that there are no ValueErrors in the `cli.py`, only `assert`. That's my personal take, you are the specialist :-)

Reply:
Yeah, you're right — I've made an assert here which will throw an AssertionError when false.